### PR TITLE
GEOMESA-1810 GeoMesa 'classpath' command is not documented

### PR DESCRIPTION
**Pull request by aheyne**
* Documented classpath command
* Documented configure command

Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/status/ClasspathCommand.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ ***********************************************************************/
+
+package org.locationtech.geomesa.tools.status
+
+import com.beust.jcommander.Parameters
+import org.locationtech.geomesa.tools.Command
+
+class ClasspathCommand extends Command {
+
+  override val name = "classpath"
```
Comment (jnh5y) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/status/ClasspathCommand.scala]: Does adding these commands interfere with bits that the bash scripts do?

Comment (aheyne) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/status/ClasspathCommand.scala]: Nope, the bash scripts intercept these before they even get to java. This just allows jcommander to report these options in the usage output.

Comment (elahrvivaz) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/status/ClasspathCommand.scala]: ah, can we add comments to that effect?

**Comment by elahrvivaz** [comment]
going to merge and I'll add comments on the classes

**Pull request merged**